insights/v1alpha1/types_insights.go

```diff
@@ -114,62 +114,6 @@ type GathererStatus struct {
 	LastGatherDuration metav1.Duration `json:"lastGatherDuration"`
 }
 
-// insightsReport provides Insights health check report based on the most
-// recently sent Insights data.
-type InsightsReport struct {
```

Comment:
So individual data gathers will now never be populated with the analysis provided by the console service? Would it not be better, if the analysis is available, to populate it on the status of the individual DataGather as well as the operator status? 

Just thinking, if I create a gather, it downloads, everything is successful, then 5 minutes later, repeat, would it not be beneficial to be able to look at the status of the 5 minute old `DataGather` and compare with the fresh one, to see if any healthchecks have been fixed or something?

Reply:
Interesting point. I didn't think about this to be honest. I only wanted to provide the latest analysis in the `insightsoperator.operator.openshift.io` CR. Thanks. So I think we can keep it and if the Insights report is not available (this info will be exposed in the conditions) then this will be just empty.

Reply:
I think it would be great to keep it if we can! The behaviour as you described is what I was thinking, if we can populate it, we do, if we can't we should be able to explain why (the report wasn't available when we went to populate it)

Reply:
I already reverted the particular commit.